src/server/scripts/BrokenIsles/EyeOfAzshara/eye_of_azshara.cpp

```diff
@@ -222,6 +223,38 @@ class aura_eoa_violent_winds_force_move : public AuraScript
     }
 };
 
+// 192737
+class aura_eoa_lightning_strikes : public AuraScript
+{
+    PrepareAuraScript(aura_eoa_lightning_strikes);
+
+    void OnPeriodic(AuraEffect const* aurEff)
+    {
+        Unit* caster = GetCaster();
+
+        if (caster->GetInstanceScript()->GetData(DATA_BOSS_DIED) >= 3)
+        {
+            Map::PlayerList const& playerList = caster->GetInstanceScript()->instance->GetPlayers();
+            std::vector<Player*> playerVec;
+            // Is there a better way to select a random player without a threat list?
+            for (auto itr = playerList.begin(); itr != playerList.end(); ++itr)
```

Comment:
std::vector<Player*> playerVec(std::begin(playerList), std::end(playerList)); should work here

Reply:
Also if this aura is used on boss encounter, you could ask boss for a target

Reply:
No, it's not used on a specific encounter